### PR TITLE
docs: correct cmd

### PR DESCRIPTION
**Pull request by benmccallum**
Awesome idea dude! 

vNext, a switch that pushes it into your spotify songs!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,4 @@
 
 Run the below command in your terminal. 
 
-`npm triple`
```
Comment (etoxin) [inline README.md]: Not a typo, i'm using a tool named npx.
https://www.npmjs.com/package/npx

File: README.md [hunk 2]
```diff
@@ -6,4 +6,4 @@
...
+`npx triplej`
```
Comment (etoxin) [inline README.md]: If you have the tool `npx` installed run command below:

```suggestion
`npx triplej`
```
Or you can install it globally with 

```
npm install -g triplej
triplej
```

File: README.md (hunk 1 above)
Comment (benmccallum) [reply README.md]: Shouldn't it be np**x** and triple**j** though?

File: README.md (hunk 2 above)
Comment (benmccallum) [reply README.md]: Yep, good to show both if anyone is using an old version of npm that doesn't have npx. Updated!

**Comment by benmccallum** [comment]
Updated it to show both usage options!

**Comment by benmccallum** [comment]
LGTM!

**Pull request closed without merging**